deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/configmap.yaml

```diff
@@ -9,6 +9,9 @@ data:
         "serverMeta": {
           "version": "{{ .Values.serverMeta.version }}",
           "turboServer": "{{ .Values.serverMeta.turboServer }}"
+        {{- if .Values.serverMeta.proxy }}
```

Comment:
Missing a comma after turboServer if proxy exists.

Reply:
thanks for catching it - pushing an update